scripts/runfabtests.sh

```diff
@@ -155,13 +155,15 @@ function compute_duration {
 }
 
 function is_excluded {
-    local e=$(echo $EXCLUDE | sed -e s/$1//)
-    if [[ "$EXCLUDE" != "$e" ]]; then
-        echo 1
-    else
-        echo 0
-    fi
-    return
+	local i
+	for i in $(echo "$EXCLUDE" | tr -s "," " "); do
+		if [[ "$i" = "$1" ]]; then
+			echo 1
+			return
+		fi
+	done
+
+	echo 0
```

Comment:
@pmmccorm I think you would want to omit the `[` `]`, since you don't want to invoke `test` here.  I.e.,
```sh
if is_excluded "$test_exe" ; then
...
```

However, I think we welcome your patch for such a modification after Ben's PR lands, since Ben is just making a straightforward correction of the logic in this function, not changing the way the function is called.